site/docs/master/on-premises.md

```diff
@@ -17,8 +17,55 @@ For example, if you use [Portworx][4] for persistent storage, you can install th
 
 If there is no native snapshot plugin available for your storage platform, you can use Velero's [restic integration][1], which provides a platform-agnostic file-level backup solution for volume data.
 
+### Air-gapped deployments
+
+In an air-gapped deployment, there is no access to the public internet, and therefore no access to public container registries.
+
+In these scenarios, you will need to make sure that you have an internal registry, such as [Harbor][5], installed and the Velero core and plugin images loaded into your internal registry.
+
+Below you will find instructions to downloading the Velero images to your local machine, tagging them, then uploading them to your custom registry.
+
+#### Preparing the Velero image
+
+First, download the Velero image, tag it for the your private registry, then upload it into the registry so that it can be pulled by your cluster.
+
+```bash
+PRIVATE_REG=<your private registry>
+VELERO_VERSION=<version of Velero you're targetting, e.g. v1.4.0>
+
+docker pull velero/velero:$VELERO_VERSION
+docker tag velero/velero:$VELERO_VERSION $PRIVATE_REG/velero:$VELERO_VERSION
+docker push $PRIVATE_REG/velero:$VELERO_VERSION
```

Comment:
are any additional steps needed here since `velero/velero` is a manifest list, which aggregates multiple arch-specific images? I don't actually know. I'm wondering if the user needs to explicitly pull/tag/push the arch-specific underlying images as well.

OK, I just tried this locally, and it looks like the relevant arch for the client machine is pulled, so if you're running linux, it'll pull the amd64 image. So that would work fine for the normal case, but if you are actually interested in the arm64 image, you'd probably have to explicitly pull it.

Reply:
I've updated these instructions, but let me know if they're unclear.